SupportClient/SupportClient/Views/AddImageView.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  AddImageView.swift
+//  SupportClient
+//
+//  Created by Cameron Russell on 3/11/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import UIKit
+
+class AddImageView: UIView {
+
+    private let addFileButton = UIButton()
+    private let noFileLabel = UILabel()
+    private var onPress: (() -> Void)?
+    
+    override var intrinsicContentSize: CGSize {
+        return UIScreen.main.bounds.size
+    }
```

Comment:
if we plan on using `UIScreen.main.bounds.size` for all `DZNEmptyDataSet` views, we can move this override to a `CustomView` class and then subclass `CustomView` so that we don't have to redeclare it for all the `DZNEmptyDataSet` views